### DISCUSSION TITLE
What's using DBUS?

### DISCUSSION THREAD

**Discussion by leseb** [kata-containers/kata-containers#5596] [Q&A] [answered]
Hi,

I noticed `/var/run/dbus` was mounted here https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml#L57-L59.

Which component is using DBUS to communicate? I could find anything in the architecture design docs, the code showed me this https://github.com/kata-containers/kata-containers/blob/a67402cc1f07094d88ae6fa11cc6675df84d18f0/src/runtime/pkg/resourcecontrol/utils_linux.go#L95.

Can someone elaborate?

Thanks in advance.

**Comment by fidencio** [comment, marked as answer]
@leseb, Kata Deploy is a daemonset used to deploy Kata Containers on a node.
When we deploy Kata Containers, we also change the containerd engine (containerd or CRI-O) configuration, and end up having to call `systemctl restart ${container_engine}`, and we have to have dbus there in order to use system, from our daemonset image.

I hope that helps.

**Reply by leseb** [reply]
Yes, it does, thanks!
